### PR TITLE
feat: add static pv link in Raddar

### PR DESCRIPTION
**Pull request by jlpedrosa**
**Description**
Same, enable static volume provisioning so pvc can be linked to existing pv.

**Checklist**

- [ ] Unit tests updated
- [ ] End user documentation updated

### Community Note

- Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

### CODE REVIEW

**Comment by M0NsTeRRR** [comment]
If you want, you can create PV links for all charts in a single PR and ask me for a review once you are ready to speed up the process :)

**Comment by jlpedrosa** [comment]
> If you want, you can create PV links for all charts in a single PR and ask me for a review once you are ready to speed up the process :)

That's better for me! thanks! I was trying to make each PR self contained.

**Comment by M0NsTeRRR** [comment]
> That's better for me! thanks! I was trying to make each PR self contained.

Your welcome, feel free to ping me once you are done.

**Comment by jlpedrosa** [comment]
@M0NsTeRRR  I'd say, let's review the two that are already open, then I will create a final one with everything pending in one go?

File: charts/radarr/values.yaml [hunk 1]
```diff
@@ -68,6 +68,9 @@ config:
       - ReadWriteOnce
     # -- Config name
     name: ""
+    # -- Name of the permanent volume to reference in the claim.
+    # Can be used to bind to existing volumes.
+    volumeName:
```
Comment (M0NsTeRRR) [inline charts/radarr/values.yaml:73]:
```suggestion
    volumeName: ""
```

File: charts/radarr/templates/persistentvolumeclaim.yaml [hunk 2]
```diff
@@ -9,6 +9,9 @@ spec:
   accessModes:
     {{- toYaml . | nindent 4 }}
   {{- end }}
+  {{- if .Values.config.persistence.volumeName }}
+  volumeName: .Values.config.persistence.volumeName
```
Comment (M0NsTeRRR) [inline charts/radarr/templates/persistentvolumeclaim.yaml:13]:
```suggestion
  volumeName: {{ .Values.config.persistence.volumeName }}
```

**Pull request merged**